pkg/networking/cilium/templater.go

```diff
@@ -112,6 +127,20 @@ func WithPolicyAllowedNamespaces(namespaces []string) ManifestOpt {
 	}
 }
 
+func (t *Templater) registryLogin(ctx context.Context, helm Helm, spec *cluster.Spec) error {
```

Comment:
looks like this "t" is not used, it can be a function instead of a method of Templater